getstarted.rst

```diff
@@ -140,26 +148,26 @@ We now install the **nio CLI**, a tool that takes care of common operations like
 Setting up a NIO Project
 ------------------------
 
-To use the nio CLI you need `git` (a distributed version control tool) and a `GitHub account <http://github.com>` with `ssh access <https://help.github.com/articles/generating-ssh-keys>`.
+To use the nio CLI you need `git <https://git-scm.com>`_ (a distributed version control tool) and a `GitHub account <http://github.com>`_ with `ssh access <https://help.github.com/articles/generating-ssh-keys>`_.
 
-To help you get started, we provide a `project template <https://github.com/nio-blocks/project_template>` which reflects the standard directory structure of a nio project. Use the nio CLI to create a new project from the template.
+After getting your github account set up and adding your ssh key to it (click the links above if you need help with this), verify that you can connect to git with: `ssh -T git@github.com`. It should return something like "Hi <YOUR_USER_NAME>! You've successfully authenticated, but GitHub does not provide shell access."
 
-Verify that you can connect to git with: `ssh git@github.com`. It should return something like "Hi YOUR_USER_NAME! You've successfully authenticated, but GitHub does not provide shell access."
 
-For Windows
-    If you have having trouble connecing, type this and then try again: ``eval $(ssh-agent -s) && ssh-add ~/.ssh/id_rsa``
+If you have having trouble connecing, type this and then try again: ``eval $(ssh-agent -s) && ssh-add ~/.ssh/id_rsa``
+
+To help you get started, we provide a `project template <https://github.com/nio-blocks/project_template>`_ which reflects the standard directory structure of a nio project. Use the nio CLI to create a new project from the template:
 
 .. code-block:: bash
 
     cd ~/nio/projects
-    nio new <project_name>
+    nio new <name_your_project>
     cd <project_name>
```

Comment:
Updates look good. Thanks. Just one comment: if the other references change from 'project_name' to 'name_your_project', then this one needs to too. I think just 'project_name' is good though. What's the reasoning for changing it?